### DISCUSSION TITLE
Anyone having issues with LORAs?

### DISCUSSION THREAD

**Discussion by halr9000** [anapnoe/stable-diffusion-webui-ux#152] [Q&A] [answered]
I'm having a ton of issues suddenly and none of my LORAs will load -- sometimes. Trying to figure out pattern, haven't gotten there yet. But when it happens, I get the below error for every LORA referenced in my prompt. TIs work fine.

```
loading Lora {path to LORA}: AttributeError
Traceback (most recent call last):
  File "D:\hal\stable-diffusion\ux\extensions\a1111-sd-webui-locon\scripts\..\..\..\extensions-builtin/Lora\lora.py", line 222, in load_loras
    lora = load_lora(name, lora_on_disk.filename)
  File "D:\hal\stable-diffusion\ux\extensions\a1111-sd-webui-locon\scripts\main.py", line 370, in load_lora
    is_sd2 = 'model_transformer_resblocks' in shared.sd_model.lora_layer_mapping
  File "D:\hal\stable-diffusion\ux\venv\lib\site-packages\torch\nn\modules\module.py", line 1614, in __getattr__
    raise AttributeError("'{}' object has no attribute '{}'".format(
AttributeError: 'LatentDiffusion' object has no attribute 'lora_layer_mapping'
```

**Comment by Zhincore** [comment]
I have the same issue!

**Reply by halr9000** [reply]
Are you using [regional prompter](https://github.com/hako-mikan/sd-webui-regional-prompter) by chance? I've uninstalled as I suspected it had something to do with things. Hard to say,.

**Reply by Zhincore** [reply]
I am not. After recent update of a1111-sd-webui-locon i now get a different error. I guess there are changes upstream not yet present in this repo?

**Reply by Zhincore** [reply]
Seems like another update of extensions fixed it

**Reply by halr9000** [reply]
> Seems like another update of extensions fixed it

mind sharing the extensions you have installed? I am still having issues

**Reply by Zhincore** [reply, marked as answer]
Sure:
- a1111-sd-webui-locon
- a1111-sd-webui-lycoris
- a1111-sd-webui-tagcomplete
- Abysz-LAB-Ext
- custom-diffusion-webui
- deforum-for-automatic1111-webui
- multi-subject-render
- sd_dreambooth_extension
- sd-dynamic-thresholding
- sd-extension-system-info
- sd_save_intermediate_images
- sd-webui-aspect-ratio-helper
- sd-webui-controlnet
- sd-webui-cutoff
- sd-webui-llul
- sd-webui-text2video
- sd-webui-vae-blessup
- seed_travel
- Stable-Diffusion-Webui-Civitai-Helper
- stable-diffusion-webui-composable-lora
- stable-diffusion-webui-pixelization
- stable-diffusion-webui-two-shot
- ultimate-upscale-for-automatic1111
- unprompted
- video_loopback_for_webui

**Reply by halr9000** [reply]
> Sure:
> - a1111-sd-webui-locon
> - a1111-sd-webui-lycoris

Weird, I have many of the same. Oh well.

Hey, you're missing one! You'll love the regional prompt control feature in this:

https://github.com/pkuliyi2015/multidiffusion-upscaler-for-automatic1111

**Reply by Zhincore** [reply]
Oh, interesting one, thank you
